sdk/appconfiguration/azure-appconfiguration-provider/README.md

```diff
@@ -72,6 +72,36 @@ config = load(endpoint=endpoint, credential=DefaultAzureCredential(), selects=se
 
 In this example all configuration with empty label and the dev label are loaded. Because the dev selector is listed last, any configurations from dev take priority over those with `(No Label)` when duplicates are found.
 
+## Dynamic Refresh
```

Comment:
Can you add a link to the public doc so that users can get detailed information on setting this up:
https://learn.microsoft.com/en-us/azure/azure-app-configuration/enable-dynamic-configuration-python